shell/services/node/node_service.cc

```diff
@@ -87,7 +87,13 @@ void NodeService::Initialize(node::mojom::NodeServiceParamsPtr params) {
   // Wrap the uv loop with global env.
   node_bindings_->set_uv_env(env);
 
-  // Load everything.
+  // LoadEnvironment should be called after setting up
+  // JavaScriptEnvironment including the microtask runner
+  // since this call will start compilation and execution
+  // of the entry script. If there is an uncaught exception
+  // the exit handler set above will be triggered and it expects
+  // both Node Env and JavaScriptEnviroment are setup to perform
+  // a clean shutdown of this process.
```

Comment:
Is this fix specific to the utility process? Do we also need to apply it elsewhere?

Reply:
Yes the other processes which setup the bindings `main` and `renderer` don't setup an exit handler so they don't have this issue. But I can reorder the calls over there as well for consistency, it has no behavior changes for those processes.

Reply:
Refs https://github.com/electron/electron/pull/34980/commits/f6b160abb490742736220374ef703bc6015a102b